packages/frontend/src/providers/__mocks__/TrackingProvider.tsx

```diff
@@ -1,17 +1,21 @@
 import { FC } from 'react';
 
-export const TrackingProvider: FC<any> = ({ children }) => {
+export const TrackingProvider: FC<React.PropsWithChildren<any>> = ({
+    children,
+}) => {
     return <>{children}</>;
 };
 
 export function useTracking() {
     return jest.fn();
 }
 
-export const TrackPage: FC<any> = ({ children }) => {
+export const TrackPage: FC<React.PropsWithChildren<any>> = ({ children }) => {
```

Comment:
check if we need `<any>` in this `TrackPage`, `TrackSection`, and `TrackingProvider`

Reply:
we dont